uiowa/uiowa
# Feature: Refactored sitenow partials in uids_base

Work on https://github.com/uiowa/uiowa/issues/1756. 

# How to test
`blt frontend`

Check on `brand.local.drupal.uiowa.edu` and make sure that the site is still styled.
